packages/suite/src/components/wallet/TransactionItem/TransactionItem.tsx

```diff
@@ -105,29 +105,34 @@ export const TransactionItem = memo(
         );
 
         const { type, targets, tokens, internalTransfers } = transaction;
+
         const isUnknown = type === 'unknown';
+        // Filter out internal transfers that are instant staking transactions
+        const filteredInternalTransfers = internalTransfers.filter(t => t.type !== 'unknown');
```

Comment:
filter using addresses as in blockchain-link-utils? this is kinda tricky

Reply:
6a6d33b28e105a1f2edf2da7a7e2fe1256ecf019